src/main/resources/templates/start.html

```diff
@@ -10,8 +10,19 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
 </head>
+
 <body>
-    <div th:replace="fragments/reusable-components :: navbar"></div>
+<div th:fragment="navbar">
+    <nav class="navbar navbar-dark bg-primary">
+        <div class="container-fluid">
+            <a class="navbar-brand">Hobby Collection</a>
+        </div>
+    </nav>
+</div>
```

Comment:
I added the head "Hobby Collection" in the layout-design theme to the webpage, so the users can easily see what application they are using and what page they are on.
Sources:
https://www.tutorialspoint.com/java/java_documentation.htm
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3